pkg/config/msp.go

```diff
@@ -670,6 +670,23 @@ func (m *MSP) validateCACerts() error {
 	if err != nil {
 		return fmt.Errorf("invalid intermediate cert: %v", err)
 	}
+	rootCertPool := x509.NewCertPool()
+	for _, cert := range m.RootCerts {
+		rootCertPool.AddCert(cert)
+	}
+	//TODO: follow the workaround that msp code use to incorporate cert.Verify()
+	for _, ic := range m.IntermediateCerts {
+		validIntermediateCert := false
+		for _, rc := range m.RootCerts {
+			err := ic.CheckSignatureFrom(rc)
+			if err == nil {
+				validIntermediateCert = true
```

Comment:
can probably break here once we've validated against a root cert

Reply:
definitely